proof/crefine/AARCH64/Retype_C.thy

```diff
@@ -7502,6 +7501,7 @@ lemma createObject_untypedRange:
             (toAPIType ty \<noteq> Some apiobject_type.Untyped \<longrightarrow> Q {} s)\<rbrace>"
   shows "\<lbrace>P\<rbrace> createObject ty ptr us dev\<lbrace>\<lambda>m s. Q (untypedRange m) s\<rbrace>"
   including no_pre
+  supply toAPIType_Some[simp del]
```

Comment:
Any thoughts on this needing to be removed from the simpset in quite a few proofs? Is it a sign that something isn't set-up quite right, or is it more to do with these being old proofs?